openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -798,6 +799,49 @@ Here is another example JWT not only proving possession of a private key but als
 }
 ```
 
+#### `ldp_vp_2.0` Key Proof Type {#ldp_vp-proof-type}
+
+The verifiable presentation MUST contain the following elements:
```

Comment:
```suggestion
The verifiable presentation as defined by [@VC_DATA_2.0] signed using Data Integrity that are used as Key Proofs MUST contain the following elements:
```

saying just 'verifiable presentation' unfortunately does not differentiate enough from key proof type JWT